atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraClusterOperations.java

```diff
@@ -81,25 +97,39 @@ private String runCql(String cql) throws IOException, InterruptedException {
                 "cqlsh",
                 "--username", CassandraContainer.USERNAME,
                 "--password", CassandraContainer.PASSWORD,
-                ThreeNodeCassandraCluster.FIRST_CASSANDRA_CONTAINER_NAME,
+                containerNames.get(0),
                 "--execute", cql);
     }
 
     private String runNodetoolCommand(String nodetoolCommand, int timeoutSeconds) throws IOException,
             InterruptedException {
+        return runNodetoolCommandOnContainer(nodetoolCommand, timeoutSeconds, containerNames.get(0));
+    }
+
+    private List<String> runNodetoolCommandOnAllContainers(String nodetoolCommand, int timeoutSeconds)
```

Comment:
Shouldnt this return void as the return value is not used anywhere. I think the intent was to follow `runNodetoolCommandOnContainer`, but this doesnt look necessary here.

Reply:
We can return void here.